lighty-core/lighty-controller-guice-di/src/test/java/io/lighty/core/controller/guice/tests/GuiceDITest.java

```diff
@@ -0,0 +1,228 @@
+package io.lighty.core.controller.guice.tests;
+
+import com.google.inject.Guice;
+import com.google.inject.Injector;
+import io.lighty.core.controller.api.LightyController;
+import io.lighty.core.controller.guice.LightyControllerModule;
+import io.lighty.core.controller.impl.LightyControllerBuilder;
+import io.lighty.core.controller.impl.config.ConfigurationException;
+import io.lighty.core.controller.impl.config.ControllerConfiguration;
+import io.lighty.core.controller.impl.util.ControllerConfigUtils;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.util.concurrent.ExecutionException;
+
+public class GuiceDITest {
+
+    private static LightyController lightyController;
+    private static TestService testService;
+
+    @BeforeClass
+    public static void init() throws ExecutionException, InterruptedException, ConfigurationException {
+        ControllerConfiguration defaultSingleNodeConfiguration = ControllerConfigUtils.getDefaultSingleNodeConfiguration();
+        LightyControllerBuilder lightyControllerBuilder = new LightyControllerBuilder();
+        lightyController = lightyControllerBuilder
+                .from(defaultSingleNodeConfiguration)
+                .build();
+        lightyController.start().get();
+
+        LightyControllerModule lightyModule = new LightyControllerModule(lightyController.getServices());
+        Injector injector = Guice.createInjector(lightyModule);
+        testService = injector.getInstance(TestService.class);
+    }
+
+    @Test
+    public void testDIDiagStatusService() {
+        Assert.assertNotNull(testService.getDiagStatusService());
+    }
+
+    @Test
+    public void testDIActorSystemProvider() {
```

Comment:
Alright, there is a ton of these. Is the expectation to add more validation, or should this really be a single test with a bunch of asserts?

Reply:
I prefer ton of small tests. Than you know exactly which DI has failed.